com.unity.render-pipelines.high-definition/Documentation~/Default-Settings-Window.md

```diff
@@ -1,40 +1,64 @@
-# Default Settings tab
+# HDRP Global Settings Window
 
-The High Definition Render Pipeline (HDRP) adds the HDRP Default Settings tab to Unity's Project Settings window. You can use this tab to set up default settings for certain features in your Project. You can:
+The High Definition Render Pipeline (HDRP) adds the HDRP Default Settings tab to Unity's Graphics Settings window. You can use this tab to set up default settings for certain features in your Project. You can:
```

Comment:
I think it's now "HDRP Settings" instead of "default"?

Reply:
yep! fix submitted 👍